wp-erp.php

```diff
@@ -424,8 +424,8 @@ public function admin_footer_text( $text ) {
         if ( 'erp' === $page ) {
             $text = sprintf(
                 /* translators: %s: review url */
-                __( 'If you like WPERP please leave us a <a href="%s" target="_blank" style="text-decoration:none">★★★★★</a> rating. Thanking you from the team of WPERP in advance!' ),
-                __( 'https://wordpress.org/support/plugin/erp/reviews/?filter=5' )
+                __( 'If you like WPERP please leave us a <a href="%s" target="_blank" style="text-decoration:none">★★★★★</a> rating. Thanking you from the team of WPERP in advance!', 'erp' ),
+                __( 'https://wordpress.org/support/plugin/erp/reviews/?filter=5', 'erp' )
```

Comment:
This is a URL, and the URL should not be translated. Instead of translating, please use 'https://wordpress.org/support/plugin/erp/reviews/?filter=5'. Also, you can use proper URL sanitization.

Reply:
I have made the correction.

In this case, what do you mean by "_you can use proper URL sanitization_"?

Thank you.